Sources/TuistDependencies/DependenciesGraph/DependenciesGraphController.swift

```diff
@@ -65,12 +70,16 @@ public final class DependenciesGraphController: DependenciesGraphControlling {
 
     public func load(at path: AbsolutePath) throws -> DependenciesGraph {
         let graphPath = self.graphPath(at: path)
+        guard FileHandler.shared.exists(graphPath) else {
+            return .none
+        }
         let graphData = try FileHandler.shared.readFile(graphPath)
 
-        let jsonDecoder = JSONDecoder()
-        let decodedGraph = try jsonDecoder.decode(DependenciesGraph.self, from: graphData)
-
-        return decodedGraph
+        do {
+            return try JSONDecoder().decode(DependenciesGraph.self, from: graphData)
+        } catch {
+            throw DependenciesGraphControllerError.failedToDecodeDependenciesGraph
```

Comment:
I'd catch the error and log it with `debug` flag

Reply:
✅ https://github.com/tuist/tuist/pull/3099/commits/0a850cb07c740f984db3d09ddd2aed99f6575e6e